.github/workflows/CI.yml

```diff
@@ -172,3 +172,19 @@ jobs:
         with:
           command: +nightly
           args: fuzz run fuzz_varint -- -runs=1
+
+  compliance:
```

Comment:
Should this be run only _after_ a merge? This currently will push to gh-pages on every PR push, I think?

Reply:
Set the pushing step to only execute if the workflow is triggered by pushing to master